benchmarks/benchmarks/optimize.py

```diff
@@ -467,25 +450,36 @@ class BenchGlobal(Benchmark):
     param_names = ["test function", "result type", "solver"]
 
     def __init__(self):
-        self.enabled = bool(slow)
-        self.numtrials = slow
+        self.enabled = is_xslow()
+        try:
+            self.numtrials = int(os.environ['SCIPY_GLOBAL_BENCH_NUMTRIALS'])
+        except (KeyError, ValueError):
+            self.numtrials = 100
 
         self.dump_fn = os.path.join(os.path.dirname(__file__), '..', 'global-bench-results.json')
         self.results = {}
 
     def setup(self, name, ret_value, solver):
         if not self.enabled:
```

Comment:
@pv @andyfaff I see 
```
    if not is_xslow():
        _enabled_functions = ['AMGM']
```
above, suggesting that `AMGM` is supposed to run even when `SCIPY_XSLOW` is not 1, yet:

```
self.enabled = is_xslow()
...
if not self.enabled:
...
   raise NotImplementedError()
```
so _none_ of the global benchmarks run when `SCIPY_XSLOW` is not 1. I think I'm reading correctly, as no benchmarks run locally when `SCIPY_XSLOW` is not 1, and there are no results for GlobalBench on Pauli's server (`AMGM` is listed, but there is no data).

Is this intentional, or should a subset of the benchmarks run when `SCIPY_XSLOW` is not 1?